smart_card_connector_app/src/_locales/en/messages.json

```diff
@@ -134,7 +134,7 @@
     }
   },
   "pcscClientHandlingUserPromptDialogMessageSuffix": {
-    "message": "is trying to access the Smart Card Connector. Would like to grant this App permission to access Smart card readers?",
+    "message": "is trying to access the Smart Card Connector. Would you like to grant this app permission to access Smart card readers?",
```

Comment:
nit: Please fix the same error in the "pcscClientHandlingUserPromptDialogMessageSuffixForUnknownApp" key below.

Reply:
Thanks, updated the "pcscClientHandlingUserPromptDialogMessageSuffixForUnknownApp" key.